### ISSUE TITLE
Can't install fresh or upgrade on Synology

### ISSUE THREAD

**Issue by tomekduda** [lloesche/valheim-server-docker#400]
I'm a Docker noob so if I'm doing something obviously wrong please be gentle.

I've installed your container over a year ago, it was time to update it. I was getting some errors so nuked the old one and started from scratch.

So dockerhub no longer hosts your builds, right? https://github.com/lloesche/valheim-server-docker/pull/398
Solution would be to to grab files from ghcr.io/lloesche/valheim-server

I'm using Synology DSM 6.2.4-25556 Update 2 (latest I can get on DS916+), latest Docker app on it and for the life of me I can't get ghcr.io to work.
If I add new registry and click to use it (pardon the Polish) it throws "registry returned an invalid value".

![image](https://user-images.githubusercontent.com/7957690/133892549-dbaf0814-9a0e-4130-b9cf-8efb3fd5e849.png)

If I click ok - the list is empty. If I go to "Image" menu and try to use "add from url" - same.

![image](https://user-images.githubusercontent.com/7957690/133892611-87f02806-ecd9-4a50-9128-85135d6d1727.png)

I Googled around a bit and it might be because GitHub doesn't support "_catalog" endpoint: 

- https://github.community/t/correct-link-to-add-to-docker-registry/160636
- https://github.community/t/github-docker-package-registry-does-not-support-catalog-endpoint/14677

So... Am I barking at wrong tree or are Synology users out of luck for a while? Any way to manually download the raw image we could import from filesystem?

I've tried to install the latest image from dockerhub but I can't get it to run, fails to download the server from Steam... As far as I can tell the filesystem permissions and firewall are ok.

**Comment by lloesche** [comment]
In commit 2d2a9ad8e516a8451b5febdee62e9ab1986ea770 I added a workflow that pushes the built Docker image to Dockerhub. So instead of Dockerhub building the image it is now being built by Github and pushed to Dockerhub. This still seems to work for non-paying Docker users. I just tested and was able to pull and start the image. Can you see if this works for you? Just `lloesche/valheim-server` should work and give you the latest image. For anyone who can I would still recommend switching over to Github Container Registry.

**Comment by tomekduda** [comment]
I experimented a bit more. SSH and `docker image ls` told me I had 2 versions of image (both from dockerhub), same name.
1 tagged latest was from 23h ago. 1 with no tag was from 7 months ago. This was despite Synology UI showing me just 1 container. 
I managed to delete the weird one, deleted everything, downloaded fresh. It starts but then I get same error as #401 

I even downloaded (ghrc one) and run the container as root over SSH. Living on the edge. Same thing.

So now it starts but fails to download the steam dedicated server. Well, at least it's consistent between dockerhub and ghcr?
